### PR TITLE
Make bridge topics configurable

### LINKED ISSUE #559: Improvement: Waku v1 <> v2 bridge topic configuration
### Problem

During [Waku v1 <> v2 `bridge` testing](https://github.com/status-im/nim-waku/issues/553) it became clear that limiting topic interest on Waku v1 and specifying the target/source `pubsubTopic` on Waku v2 will be a useful addition to the `bridge` configuration. This way noise can be limited and the `bridge` only configured for those topics that are earmarked for v1-v2 bridging.

### Acceptance Criteria

- [x] V1 topic interest can be configured on `bridge`, and added to via API calls
- [x] Bridging `pubsubTopic` can be configured

### PR DESCRIPTION
**Pull request by jm-clius**
This PR closes #559 

It allows (optional) topic configuration on the v1 <> v2 bridge, namely:
1. Configuring the v2 pubsubTopic to bridge to/from. This defaults to the usual `/waku/2/default-waku/proto`.
2. Setting a v1 topic interest, which can be added to via an RPC API call. By default the bridge gets configured as a v1 node with full bloom filter so that it will receive and forward all messages.

The ability to configure topics on the `bridge` helps to limit noise and to configure the `bridge` for test environments.

### CODE REVIEW

**Review by staheri14 — APPROVED** [review]
Looks good to me! 
just some minor comments re some namings.

File: waku/common/config_bridge.nim [hunk 1]
```diff
@@ -97,6 +97,11 @@ type
       desc: "PoW requirement of Waku node.",
       defaultValue: 0.002
       name: "waku-pow" .}: float64
+    
+    wakuTopicInterest* {.
+      desc: "Run as node with a topic-interest",
```
[the remark this answers is not in the document]
Comment (oskarth) [reply waku/common/config_bridge.nim:102]: Agree, I'd make v1 explicit in parameter name too to make it unambiguous

**Review by oskarth — APPROVED** [review]

File: waku/common/config_bridge.nim (hunk 1 above)
[the remark this answers is not in the document]
Comment (kdeme) [reply waku/common/config_bridge.nim:102]: > If I am not mistaken, topic-interest indicates pubsub topic interest for v1 node, right?
> If so, then I suggest something like this `run as a waku v1 node with a pubsub topic interest`

In waku v1, topic-interest indicates actually topics which are a mix of the v2 pubsub and content topics.
That is, they are directly related to content, but you can still filter them out by sort of "not subscribing". This is done by bloom filter or by the topic-interest (never both).

In that sense, a bridge would usually always set a full bloom filter, to be able to forward all content to waku v2, which is why I never added it initially in the configuration. I can understand however that for testing reasons it is a handy option to have.

**Review by kdeme — APPROVED** [review]

File: waku/common/wakubridge.nim [hunk 2]
```diff
@@ -40,6 +39,7 @@ type
     nodev1*: EthereumNode
     nodev2*: WakuNode
     seen: seq[hashes.Hash] # FIFO queue of seen WakuMessages. Used for deduplication.
+    pubsubTopic: wakunode2.Topic # Pubsub topic to bridge to/from
```
[the remark this answers is not in the document]
Comment (jm-clius) [reply waku/common/wakubridge.nim]: Thanks. Done.

File: waku/common/config_bridge.nim (hunk 1 above)
[the remark this answers is not in the document]
Comment (jm-clius) [reply waku/common/config_bridge.nim]: Thanks all. Have improved naming and description in 1dee057

**Pull request merged**